### PR TITLE
Edit error message and success message 

### LINKED ISSUE #97: Standardise error messages

### PR DESCRIPTION
**Pull request by Jayne1010**
Closes #97 and #103
This PR includes:

- Standardising error message for commands 
- Leaving a space between each parameters when commands are successfully executed.

### CODE REVIEW

**Review by tengcharmaine — DISMISSED** [review]
LGTM!

File: src/main/java/seedu/address/logic/commands/EditCommand.java [hunk 1]
```diff
@@ -41,21 +41,28 @@ public class EditCommand extends Command {
     public static final String COMMAND_WORD = "edit";
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the details of the person identified "
-            + "by the index number used in the displayed person list. "
+            + "by the index number used in the displayed person list. \n"
```
Comment (tengcharmaine) [inline src/main/java/seedu/address/logic/commands/EditCommand.java]: i think this one should be by the person's ic number?

**Review by tengcharmaine — COMMENTED** [review]
I have 1 comment for the edit command but other than that LGTM!

File: src/main/java/seedu/address/logic/commands/AddNoteCommand.java [hunk 2]
```diff
@@ -22,17 +22,18 @@ public class AddNoteCommand extends Command {
...
 
     public static final String MESSAGE_USAGE = COMMAND_WORD + ": Edits the note of the person whose profile matches "
-            + "the specified IC (case-insensitive). "
-            + "Existing remark will be appended by default. "
-            + "To replace the original note, add -replace at the end of your command. "
-            + "E.g. addnote i/S0123456Q n/Diabetes -replace\n"
+            + "the specified IC (case-insensitive). \n"
+            + "Existing remark will be appended by default. \n"
             + "Parameters: "
             + PREFIX_IC + "IC "
```
Comment (jiahui0309) [inline src/main/java/seedu/address/logic/commands/AddNoteCommand.java]: maybe we can standardise with editcommand and put "Parameters: IC (must be a valid identity card number) "?

**Review by jiahui0309 — CHANGES_REQUESTED** [review]
LGTM other than a minor comment!

File: src/main/java/seedu/address/logic/commands/EditCommand.java (hunk 1 above)
Comment (Jayne1010) [reply src/main/java/seedu/address/logic/commands/EditCommand.java]: okay fixed, let me know if there are other typos

File: src/main/java/seedu/address/logic/commands/AddNoteCommand.java (hunk 2 above)
Comment (Jayne1010) [reply src/main/java/seedu/address/logic/commands/AddNoteCommand.java]: yep! sorry overlooked that :)

**Review by tengcharmaine — APPROVED** [review]
LGTM!

**Pull request merged**